components/atoms/Swap.tsx

```diff
@@ -1,5 +1,6 @@
 import { useState } from "react";
-import { Box, Button, Flex, HStack, Image, Input, Select, Spacer, Text, VStack } from "@chakra-ui/react";
+import { Box, Button, createIcon, Flex, HStack, IconButton, Image, Input, Select, Spacer, Text, VStack } from "@chakra-ui/react";
+import { HiOutlineSwitchVertical } from 'react-icons/hi';
```

Comment:
we are delivering highly custom designed work here, imo we should stick to figma vectors and keep our dependencies as small as possible in this case